rma_sale/controllers/sale_portal.py

```diff
@@ -25,12 +25,13 @@ def request_rma(self, order_id, access_token=None, **post):
             mapped_vals.setdefault(row, {}).update({field_name: value})
         line_vals = [(0, 0, vals) for vals in mapped_vals.values()]
         # Create wizard an generate rmas
-        location_id = order_obj.browse(order_id).warehouse_id.rma_loc_id.id
+        order = order_obj.browse(order_id).sudo()
+        location_id = order.warehouse_id.rma_loc_id.id
         wizard = wizard_obj.with_context(active_id=order_id).create({
             'line_ids': line_vals,
             'location_id': location_id
         })
-        rma = wizard.create_rma(from_portal=True)
+        rma = wizard.sudo().create_rma(from_portal=True)
```

Comment:
The problem with this is that the create user will be system, not the proper user. Fortunately, this won't happen in v13, right?

Reply:
Fortunately, this won't happen in v13, right?

Why not?

Reply:
`sudo()` only change the permissions, not the uid.

Reply:
:+1: